.github/workflows/discover-lean-pr-testing.yml

```diff
@@ -100,8 +100,13 @@ jobs:
             
             # Check if the diff contains files other than the specified ones
             if echo "$DIFF_FILES" | grep -v -e 'lake-manifest.json' -e 'lakefile.lean' -e 'lean-toolchain'; then
-                # If it does, add the branch to RELEVANT_BRANCHES
-                RELEVANT_BRANCHES="$RELEVANT_BRANCHES"$'\n- '"$BRANCH"
+                # Extract the PR number and actual branch name
+                PR_NUMBER=$(echo "$BRANCH" | grep -oP '\d+$')
+                ACTUAL_BRANCH=$(echo "$BRANCH" | sed 's|origin/||')
```

Comment:
```suggestion
                ACTUAL_BRANCH=${BRANCH#origin/}
```
I asked Claude what the shellcheck suggestion meant, and it said to try this... 🤷‍♀️

Reply:
I think that this means to remove `origin/` from the start of the variable `$BRANCH`. It may differ from `sed` since the regex does not have `^origin/^.  Though, in context, I actually think that it is better to remove `origin/` *only* when it starts the string anyway.